android/app/src/main/java/mqttController/MqttClient.java

```diff
@@ -65,4 +68,8 @@ public void publish(String topic, String message, int qos, IMqttActionListener p
             e.printStackTrace();
         }
     }
+
+    public void switchServer(String newServerUrl){
+        this.mMqttAndroidClient = new MqttAndroidClient(this.context, newServerUrl, clientId);
+    }
```

Comment:
This method does not seem to be in use anymore after using separate local and public MQTT clients. I think it could be removed.

Reply:
That is a good point. I removed it.